public/app/features/alerting/unified/useRouteGroupsMatcher.ts

```diff
@@ -51,8 +51,6 @@ export function useRouteGroupsMatcher() {
   useEffect(() => {
     const { disposeWorker } = loadWorker();
     return disposeWorker;
-
-    return () => null;
```

Comment:
@konrad147 any particular reason why we had an empty callback here? I'm pretty sure this is a no-op?

Reply:
Looks like it might have been a mistake, as this was unreachable code from the previous `return`?

I tried enabling `no-unreachable` rule in eslint and this was the only occurrence (I think), so perhaps we could turn that rule on?

Reply:
I'm surprised it hasn't been enabled yet? Unreachable code like this one is usually pretty easy to detect.

Reply:
Yep I was also surprised it wasn't already on! But nice that there wasn't anything else that snuck through, it seems

Reply:
It looks like there used to be a condition there and we forgot to remove `return () => null` when removing the condition https://github.com/grafana/grafana/commit/323ee7c38ceb18b8e71c780d797fabac7041a673?diff=split&w=0#diff-d3a38f76f1986d3ec9428731fffeec62acdf42df317e8df30ed54af3c825ca19L61-L71

I think it's safe to remove the line